frontend/app/component/error/Error.types.ts

```diff
@@ -0,0 +1,13 @@
+export enum ErrorAction {
+  None = "none",
+  Back = "back",
+}
+
+export class NotFoundError extends Error {
```

Comment:
i have no suggestion, but this is not a type 😬

Reply:
Although its not a pure Typescript type, a TypeScript class definition creates a type from itself - so maybe at least half the correct place 😅